test/Elastica/SnapshotTest.php

```diff
@@ -12,7 +12,7 @@
  */
 class SnapshotTest extends Base
 {
-    private const SNAPSHOT_PATH = '/usr/share/elasticsearch/data/';
+    private const SNAPSHOT_PATH = '/usr/share/elasticsearch/repository/';
```

Comment:
This is required as the storage location of the repository changed: we use 2 different volumes for the ES data and the repository (the latter is shared among all nodes)